backend/pkg/config/ratelimits/config.go

```diff
@@ -24,6 +24,14 @@ import (
 	"github.com/mendersoftware/mender-server/pkg/redis"
 )
 
+type ConfigDisabledError struct {
+	Path string
+}
+
+func (err *ConfigDisabledError) Error() string {
+	return fmt.Sprintf("configuration %q disabled", err.Path)
```

Comment:
this is %q and we want it because we need it? if not, the simple string concat would be much faster here.